Make DocumentCloud viewer more responsive

rosaliechan opened the pull request:
This is a fix to this issue: https://github.com/NUKnightLab/TimelineJS3/issues/330
I set a minimum width so that the arrow does not overlap with the full screen.
However, the issue when I did that was that the text on the right would sometimes go under the documentcloud embed. To resolve this, I changed skinny_size from 650 to 770. Thoughts on this? @scott2b

zachwise commented:
Changing `skinny_size` will have many unwanted effects on UX and many existing timelines. I STRONGLY suggest not changing it.

rosaliechan commented:
Thanks for letting me know. I have changed skinny_size back to the values they were before. Instead, in TL.Media.DocumentCloud.js, in the _updateMediaDisplay function, I added a condition for if the width is between skinny_size and medium_size. This way, the DocumentCloud arrow does not overlap with the fullscreen, and the text on the right does not run over or under the document. @scott2b

rosaliechan commented:
Closing because this is not finished. Waiting to hear back from DocumentCloud support because we might not be able to control this.
